hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestOzoneFSWithObjectStoreCreate.java

```diff
@@ -386,6 +387,47 @@ public void testListKeysWithNotNormalizedPath() throws Exception {
     checkKeyList(ozoneKeyIterator, keys);
   }
 
+  @Test
+  public void testDoubleSlashPrefixPathNormalization() throws Exception {
+    OzoneVolume ozoneVolume =
+        client.getObjectStore().getVolume(volumeName);
+    OzoneBucket ozoneBucket = ozoneVolume.getBucket(bucketName);
+    String doubleSlashKey = "//dir1/key1";
+    String normalizedDoubleSlashKey = "dir1/key1";
+    String tripleSlashKey = "///dir2/key2";
+    String normalizedTripleSlashKey = "dir2/key2";
```

Comment:
To avoid duplication, the test can be parameterized with number of slashes to add as prefix.